packages/metrics-handler/index.js

```diff
@@ -0,0 +1,57 @@
+function responseHelper(body) {
```

Comment:
this name is not communicating the intention very well. Maybe let's call it `createOkResponse` or something similar?